### PR TITLE
[Merged by Bors] - Improve quickstart guide

### PR DESCRIPTION
**Pull request by sbernauer**
## Description

*Please add a description here. This will become the commit message of the merge request later.*

## Review Checklist
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/ROOT/pages/demos/trino-taxi-data.adoc [hunk 1]
```diff
@@ -16,6 +23,36 @@ You can see the deployed products as well as their relationship in the following
...
+                                   coordinator-https   https://172.18.0.3:30141                                       
+                                                                                                                      
+ minio     minio-trino  default    http                http://172.18.0.3:31062   Third party service                  
+                                   console-http        http://172.18.0.3:30503   Admin user: root, password: rootroot
+----
+
+[NOTE]
+====
+When a product instance has not finished starting yet, the service will have no endpoint.
+Starting all of the product instances might take an considerable amount of time depending on your Internet connectivity.
```
Comment (maltesander) [inline docs/modules/ROOT/pages/demos/trino-taxi-data.adoc:52]:
```suggestion
Starting all of the product instances might take an considerable amount of time depending on your internet connectivity.
```

**Review by maltesander — APPROVED** [review]
LGTM.

File: docs/modules/ROOT/pages/demos/trino-taxi-data.adoc (hunk 1 above)
Comment (sbernauer) [reply docs/modules/ROOT/pages/demos/trino-taxi-data.adoc:52]: https://en.wikipedia.org/wiki/Capitalization_of_Internet lol

**Pull request closed without merging**